src/core/Table/utils.ts

```diff
@@ -6,14 +6,20 @@ import { ColumnInstance } from 'react-table';
 
 export const getCellStyle = <T extends Record<string, unknown>>(
   column: ColumnInstance<T>,
+  isTableResizing = false,
```

Comment:
maybe make it mandatory to always pass it from table instance?

Reply:
Changed